docs/testplan/GCU-Dynamic-ACL-testplan.md

```diff
@@ -268,9 +308,61 @@ DYNAMIC_ACL_TABLE | DYNAMIC_ACL_TABLE_TYPE | {vlan port 1} | DYNAMIC_ACL_TABLE_T
 
 **Additional Checks**
 + Check that results of “show acl rule DYNAMIC_ACL_TABLE RULE_1” and “show acl rule DYNAMIC_ACL_TABLE RULE_2” match the following output:
-  + DYNAMIC_ACL |_TABLE	RULE_1 | 9999 | FORWARD |  DST_IP: 103.23.2.1/32
+  + DYNAMIC_ACL_TABLE |	RULE_1 | 9999 | FORWARD |  DST_IP: 103.23.2.1/32
   + DYNAMIC_ACL_TABLE | RULE_2 | 9998 | FORWARD | DST_IPV6: 103.23.2.1::1/128
 
+### Create DHCP Forward Rule
+
+**Json Patch:**
+
+    [
+        {
+            "op": "add",
+            "path": "/ACL_RULE",
+            "value": {
+                "DYNAMIC_ACL_TABLE|DHCP_RULE": {
+		    "IP_PROTOCOL": "UDP",
+                    "L4_DST_PORT: "67",
+                    "PRIORITY": "9999",
+                    "PACKET_ACTION": "FORWARD"
+                }
+            }
+        }
```

Comment:
The DHCPv6 is using different port. So we need separated rules for DHCPv4 and DHCPv6.